cli/cmd/inject.go

```diff
@@ -671,6 +681,17 @@ func generateReport(injectReports []injectReport, output io.Writer) {
 		output.Write([]byte(fmt.Sprintf("%s%s -- %s %s \"hostNetwork: true\"\n", hostNetworkPrefix, warnStatus, strings.Join(hostNetwork, ", "), verb)))
 	}
 
+	sidecarPrefix := fmt.Sprintf("%s%s", sidecarDesc, getFiller(sidecarDesc))
+	if len(sidecar) == 0 {
+		output.Write([]byte(fmt.Sprintf("%s%s\n", sidecarPrefix, okStatus)))
+	} else {
+		verb := "uses"
+		if len(sidecar) > 1 {
+			verb = "use"
+		}
+		output.Write([]byte(fmt.Sprintf("%s%s -- %s %s known sidecar\n", sidecarPrefix, warnStatus, strings.Join(sidecar, ", "), verb)))
```

Comment:
TIOLI you could use the text `known sidecar detected in strings.Join(sidecar, ", ")` (or `known sidecar already present in ...`) to avoid having the use/uses.